api/cf/cloudfoundry.ts

```diff
@@ -188,14 +88,9 @@ export async function getSpace(guid: string): Promise<Response> {
   return await cfRequest('/spaces/' + guid, 'get');
 }
 
-export async function getSpaces(orgGuids?: string[]): Promise<Response> {
-  if (orgGuids && orgGuids.length > 0) {
-    const params = new URLSearchParams({
-      organization_guids: orgGuids.join(','),
-    });
-    return await cfRequest('/spaces?' + params.toString());
-  }
-  return await cfRequest('/spaces');
+export async function getSpaces({ ...args }: GetSpaceArgs): Promise<Response> {
```

Comment:
[blocking suggestion] Since every param is optional for a `GetSpaceArgs` object, try adding a default argument of `{}`. That way, we don't have to pass an empty object and can just call `getSpaces()` with no arg
```
export async function getSpaces({ ...args }: GetSpaceArgs = {}): Promise<Response> {
```